packages/devtools_app/lib/src/flutter/controllers.dart

```diff
@@ -66,9 +66,9 @@ class Controllers extends StatefulWidget {
   _ControllersState createState() => _ControllersState();
 
   static ProvidedControllers of(BuildContext context) {
-    final _InheritedProvider inherited =
-        context.dependOnInheritedWidgetOfExactType(aspect: _InheritedProvider);
```

Comment:
aspect is only for use with InheritedModel, not regular InheritedWidgets.

Reply:
ah i see, sorry for this one :-)